packages/dotcom-build-base/package.json

```diff
@@ -29,12 +30,16 @@
     "webpack": "^4.39.2"
   },
   "engines": {
-    "node": ">= 12.0.0"
+    "node": ">= 12.0.0",
+    "npm": "7.x || 8.x"
```

Comment:
this makes sense to me, even though we're using npm 8 for the circleci build, npm version 7 is the minimum requirement for migrating components away from bower